CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Add query for initialized extensions ([#5658](https://github.com/opensearch-project/OpenSearch/pull/5658))
 - Add update-index-settings allowlist for searchable snapshot ([#5907](https://github.com/opensearch-project/OpenSearch/pull/5907))
 - Replace latches with CompletableFutures for extensions ([#5646](https://github.com/opensearch-project/OpenSearch/pull/5646))
+- Add support to disallow search request with preference parameter with strict weighted shard routing([#5874](https://github.com/opensearch-project/OpenSearch/pull/5874))
```

Comment:
This entry is now in the wrong section [on the main branch](https://github.com/opensearch-project/OpenSearch/blob/main/CHANGELOG.md?plain=1#L13). @Bukhtawar @anshu1106 Can one of you fix that? (Note the entry being in the wrong spot is likely one of the conflicts that caused the cherry-pick to fail the automated backport)

Reply:
Sure. Created a PR to fix this https://github.com/opensearch-project/OpenSearch/pull/6057